extensions/wikia/EmbeddableDiscussions/models/DiscussionsThreadModel.class.php

```diff
@@ -56,41 +56,17 @@ private function categoryLookup( $category, $rawData ) {
 	}
 
 	private function getRequestUrl( $showLatest, $limit, $category ) {
-		global $wgDevelEnvironment;
-
 		$sortKey = $showLatest ? self::SORT_LATEST : self::SORT_TRENDING;
 		$categoryId = $this->getCategoryId( $category );
 		$categoryKey = $categoryId ? '&forumId=' . $categoryId : '';
 
-		if ( empty( $wgDevelEnvironment ) ) {
-			return self::DISCUSSIONS_API_BASE .
-				"$this->cityId/threads?sortKey=$sortKey&limit=$limit&viewableOnly=false" . $categoryKey;
-		}
-
-		return self::DISCUSSIONS_API_BASE_DEV .
-			"$this->cityId/threads?sortKey=$sortKey&limit=$limit&viewableOnly=false" . $categoryKey;
+		return "/$this->cityId/threads?sortKey=$sortKey&limit=$limit&viewableOnly=false" . $categoryKey;
 	}
 
 	private function getUpvoteRequestUrl() {
-		global $wgDevelEnvironment;
-
-		if ( empty( $wgDevelEnvironment ) ) {
-			return "/$this->cityId/votes/post/";
-		}
-
 		return "/$this->cityId/votes/post/";
 	}
 
-	private function getBaseUrl() {
-		global $wgDevelEnvironment;
-
-		if ( empty( $wgDevelEnvironment ) ) {
-			return self::DISCUSSIONS_API_BASE;
-		}
-
-		return self::DISCUSSIONS_API_BASE_DEV;
-	}
-
 	public function getData( $showLatest, $limit, $category ) {
 		$sortKey = $showLatest ? self::SORT_LATEST_LINK : self::SORT_TRENDING_LINK;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)